nativeauthenticator/nativeauthenticator.py

```diff
@@ -33,20 +46,39 @@ def authenticate(self, handler, data):
         if user.is_authorized and user.is_valid_password(data['password']):
             return data['username']
 
-    def get_or_create_user(self, username, password):
-        user = User.find(self.db, username)
-        if not user:
-            user = User(name=username, admin=False)
-            self.db.add(user)
+    def is_password_common(self, password):
+        common_credentials_file = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            'common-credentials.txt'
+        )
+        if not self.COMMON_PASSWORDS:
+            with open(common_credentials_file) as f:
+                self.COMMON_PASSWORDS = f.read().splitlines()
+        return password.lower() in self.COMMON_PASSWORDS
```

Comment:
Does COMMON_PASSWORDS only contain lowercase characters?

Reply:
Indeed, it doesn't. I removed this `lower()` method :)